how-to-pitch-post-supply/readme.md

```diff
@@ -0,0 +1,49 @@
+
+We offer a solution to a problem. As long as we are able to explain how we solve the problem, those that have the problem will see the value in what we do.
+
+The product should sell itself, as long as people know what the product is. If not, we need to redefine the product, not try to sell harder.
+
+## The basics of what we do and why we do it.
+
+
+### Problem
+
+Social media is mostly used to influence the way people perceive the sender, creating large amounts of aggressive and pushy content that has no real value for the receiver. This is a waist of resources and drowns out the content that is actually valuable for the receiver.
+
+### Solution
+
+We are convinced that you can have effective communication by using it as a tool to achieve the goals of your company.
+
+The best way to do this is by enabling your audience to be part of the solution to the problem your company is trying to solve.
+
+This way you show the importance of what you do, you validate your expertise, you are valuable to your audience and you get closer to achieving the goals of your company.
+
+### Our product
+
+We teach you how to do this trough our website. If you do not have the time to do it yourself, we can do it for you trough our application.
+
+**Our website** is the place where we publicly store all the lessons we have learned about reaching goals trough good content. Our website is always the first place where we look when researching for new articles. By making it public we hope to enable others to do this as well.
+
+**Our application** allows the user to manage a multidisciplinary journalistic team that collaborates closely together to create articles that achiever the goals of the user. Using the online interface, the user can decide the goal of the publications, commission proposed articles and approve them when they're done.
+
+Apart from creating the right content, the main focus of the application is to enable users to have and maintain a steady flow of content with the least amount of effort.
+
+**Our work Method** enables us to have team members work on the same articles simultaneously from their own strength while covering the shortcomings of others. This enables us to create diverse articles that are not subjected to the knowledge and experience of one writer allowing us to be able to scale and adapt where necessary. We call this Agile Journalism.
+
+## Our goal
+
+Our main goal is to stimulate the use of content as tool to reach the goals of companies to reduce the amount of pushy and empty content, and increase the amount of communication that is actually valuable for the receiver.
+
+## How to tell this story
+
+When telling the story for the first time the most important thing is to tell only what is necessary and when it is necessary, in such a way that there is a logical transition.
+
+ This is something you will only learn to practice. So don't worry if you don't get it right the first 50 times.
```

Comment:
This is something you will only learn to practice. So don't worry if you don't get it right the first 50 times.-->This is something you will only learn with practice, so don't worry if you don't get it right the first 50 times.